unstructured/partition/common.py

```diff
@@ -460,7 +460,8 @@ def convert_to_bytes(
 
 
 def convert_ms_office_table_to_text(
-    table: Union["docxtable", "pptxtable"], as_html: bool = True
+    table: Union["docxtable", "pptxtable"],
+    as_html: bool = True,
```

Comment:
Is Black doing this? Or just a project convention? If I'm running the wrong Black version or config I'd like to fix it :)

Reply:
My black configuration was misconfigured and I think set some of these. I can fix.

Reply:
Looks like these adjustments are in line with the results of `make tidy` and `make check`